### PR TITLE
DDCNOS-1908: Improve accessibility of language selector navigation fo…

### PR DESCRIPTION
**Pull request by vajarova**
…r screen readers

### CODE REVIEW

File: resources/templates/mustache/production/govuk-template.mustache.html [hunk 1]
```diff
@@ -196,15 +196,15 @@
...
+                                                        <span class="faded-text--small bold"  >| Cymraeg</span>
                                                     </li>
                                                 {{/isWelsh}}
 
                                                 {{^isWelsh}}
                                                     <li>
-                                                        <span class="faded-text--small"><strong class="bold">English |</strong></span>
-                                                        <a id="switchToWelsh" lang="cy" href="{{cyUrl}}" data-journey-click="link - click:lang-select:Cymraeg"><small><strong class="bold">Cymraeg</strong></small></a>
+                                                        <span class="faded-text--small bold" aria-hidden="true">English |</span>
+                                                        <a id="switchToWelsh" lang="cy" href="{{cyUrl}}" data-journey-click="link - click:lang-select:Cymraeg" aria-label="Dewiswch Cymraeg">Cymraeg</a>
```
Comment (danewatts) [inline resources/templates/mustache/production/govuk-template.mustache.html:207]: You dont have the quotes this time

File: resources/templates/mustache/production/govuk-template.mustache.html [hunk 2]
```diff
@@ -196,15 +196,15 @@
                                             {{#cyUrl}}
                                                 {{#isWelsh}}
                                                     <li>
-                                                        <a id="switchToEnglish" lang="en" href="{{enUrl}}" data-journey-click="link - click:lang-select:English"><small><strong class="bold">English</strong></small></a>
-                                                        <span class="faded-text--small"><strong class="bold">| Cymraeg</strong></span>
+                                                        <a id="switchToEnglish" lang="en" href="{{enUrl}}" data-journey-click="link - click:lang-select:English" aria-label="Select English">English</a>
+                                                        <span class="faded-text--small bold"  >| Cymraeg</span>
```
Comment (danewatts) [inline resources/templates/mustache/production/govuk-template.mustache.html]: You dont have `aria-hidden="true"`

**Review by danewatts — CHANGES_REQUESTED** [review]

File: resources/templates/mustache/production/govuk-template.mustache.html (hunk 1 above)
Comment (vajarova) [reply resources/templates/mustache/production/govuk-template.mustache.html:207]: They have been removed from frontend-template as well but it was a separate pr that fixed failing tests and removed quotes. sorry

Comment (danewatts) [reply resources/templates/mustache/production/govuk-template.mustache.html:207]: Ahh, I didnt check master branch, I only checked that PR. My bad.

**Review by danewatts — APPROVED** [review]

**Pull request merged**